lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +26,63 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class MobileAppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    IOS = "ios"
+    ANDROID = "android"
+    MOBILE_PLATFORM = (
+        (IOS, "IOS"),
+        (ANDROID, "ANDROID"),
+
+    )
+    KEY_FIELDS = ('platform', )  # mobile platform is unique
+    platform = CharField(
+        max_length=50,
+        choices=MOBILE_PLATFORM,
+        blank=False,
+        help_text="mobile device platform")
+    latest_version = TextField(blank=False, help_text="Latest available version for app in X.X.X format")
+    min_supported_version = TextField(blank=False, help_text="Min supported version for app in X.X.X format")
+    next_min_supported_version = TextField(
```

Comment:
@nasthagiri @wajeeha-khalid noted that we need to differentiate between 4 cases. `min_supported_version` is used to check if 1 is true and `next_min_supported_version` checks which of 2 or 3 is true.

1. Request is from a version which is not supported (result: return an error).
2. Request is from a version which is deprecated and the user needs to upgrade in the near future. (result: tell user they need to upgrade)
3. Request is from a supported but not latest version (result: tell user there is a newer version available).
4. Request is from the latest version.

Do you think there is a way of differentiating between these 4 cases if we remove this field?